documentation/users-guide/source/reference.rst

```diff
@@ -391,5 +391,24 @@ Test Execution
    :class:`<test-runner>` based on the command-line options and then
    calls :func:`run-tests` with the runner and *suite-or-test*.
 
+.. function:: test-option
+
+   Return an option value passed on the test-application command line.
+
+   :signature: test-option *name* #key *default* => *value*
+   :parameter name: An instance of type :drm:`<string>`.
```

Comment:
This should mention ``:paramater #key default: An instance of type :drm:`<string>`.``